package.json

```diff
@@ -6,12 +6,17 @@
     "@types/node": "^20.11.20",
     "concurrently": "^8.2.2",
     "nodemon": "^3.1.0",
-    "typescript": "^5.3.3"
+    "typescript": "^5.3.3",
+    "prettier": "^3.2.5",
+    "eslint": "^8.48.0"
   },
   "scripts": {
     "install-all": "npm i --verbose",
     "clean": "rm -rf package-lock.json **/package-lock.json **/**/package-lock.json dist **/dist **/**/dist node_modules **/node_modules **/**/node_modules tsconfig.tsbuildinfo **/tsconfig.tsbuildinfo **/**/tsconfig.tsbuildinfo",
-    "build": "tsc --build --verbose"
+    "build": "tsc --build --verbose",
+    "lint": "npx eslint --ext .ts ./",
```

Comment:
Linting doesn't seem to work for me. I get a `ESLint couldn't find the config "standard-with-typescript" to extend from. Please check that the name of the config is correct` It seems there is a difference in the `.eslint.json` between the directories. Do you have a best practice document somewhere that we can use and make the standard in citrine?

Reply:
@ChrisWeissmann you did `npm i` right? because I think in eslint we extend this standard from other npm packages

Reply:
yea, I have run `npm i` The problem is in the `01_Data` 
> The config "standard-with-typescript" was referenced from the config file in "/Users/christian/Developer/citrineos-core/01_Data/.eslintrc.json".

In there we have `"extends": "standard-with-typescript",` while in the others we don't have that. 
There we have: `"extends": "eslint:recommended"` and in some places we have  `extends: ['eslint:recommended', 'plugin:@typescript-eslint/recommended'],`

Probably worth having one "right" config.